### DISCUSSION TITLE
Neovide freeze if there is a message or error at startup

### DISCUSSION THREAD

**Discussion by azzamsa** [neovide/neovide#2254] [Q&A] [answered]
Hi 👋🏼

If there is an error at startup, Neovide freezes. I can't type anything. 
I need to do `killall neovide` and then fix the error using plain Neovim.

![image](https://github.com/neovide/neovide/assets/17734314/e9e463cb-c817-4764-8708-de97f58ad3c8)

[neovide_rCURRENT.log](https://github.com/neovide/neovide/files/13877360/neovide_rCURRENT.log)

```
NVIM v0.9.4
neovide 0.12.1
```

```
- **OS Name:**                                     Fedora Linux 39 (Workstation Edition)
- **OS Type:**                                     64-bit
- **GNOME Version:**                               45.2
- **Windowing System:**                            Wayland
- **Kernel Version:**                              Linux 6.6.9-200.fc39.x86_64
```

**Comment by fredizzimo** [comment, marked as answer]
This is fixed on main, we will release a new version soon, but until then you can build from source.

**Reply by azzamsa** [reply]
I checked every day, and no one (from 11.2k stargazers) opened an issue about this. So I thought it *must* be from my config.

Thank you for maintaining Neovide! 🍦

**Reply by azzamsa** [reply]
Would you like to share related commits that fix this issue?

**Reply by fredizzimo** [reply]
Yes, this PR fixed it https://github.com/neovide/neovide/pull/2241
